src/site/content/en/blog/backdrop-filter/index.md

```diff
@@ -39,6 +39,20 @@ Historically, these techniques were difficult to implement on the web, requiring
   </figcaption>
 </figure>
 
+## Browser support
+
+{% BrowserCompat 'css.properties.backdrop-filter' %}
+
+For performance reasons, fall back to an image instead of a polyfill when `backdrop-filter` isn't supported. The example below shows this.
+
+```css
+@supports not (backdrop-filter: none) {
+  .background {
+    background-image: blurred-hero.png;
+  }
+}
+```
```

Comment:
```suggestion
```css
@supports (backdrop-filter: none) {
	.background {
		backdrop-filter: blur(10px);
	}
}

@supports not (backdrop-filter: none) {
  .background {
    background-image: blurred-hero.png;
  }
}
```
```